project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -110,6 +125,26 @@ public static Builder builder() {
      */
     public abstract Builder setPriceLevel(int priceLevel);
 
+    /**
+     * @param googleUrl the URL of the place’s google page.
+     * @return a Place builder that enables to continue building
+     */
+    public abstract Builder setGoogleUrl(String googleUrl);
+
+    /**
+     * @param placeId the place's placeId as defiened on Google Places.
+     * @return a Place builder that enables to continue building
+     */
+    public abstract Builder setPlaceId(String placeId);
+
+    /**
+     * @param businessStatus the place's status, represented by one of the following Strings:
+     * "OPERATIONAL", "CLOSED_TEMPORARILY", "CLOSED_PERMANENTLY" and null if the information
+     * isn't available.
+     * @return a Place builder that enables to continue building
+     */
+    public abstract Builder setBusinessStatus(String businessStatus);
```

Comment:
Have you considered making this an `enum` instead of String? This way you'll always have one of multiple available values, and won't run into any surprises and won't need to do any validation.

Reply:
Changed it to enum, it does handel the business status better. I don't have exprience with using enums so let me know if this is really the way to do it.